### DISCUSSION TITLE
Is there interest in v-table or simple-v-table allowing editing of cells?

### DISCUSSION THREAD

**Discussion by esnyder** [day8/re-com#249] [Ideas]
Hi all,

I am starting work on a project that requires some sort of editable-cells-spreadsheet'ish-interface component. I've only recently found re-frame and re-com and from my (relatively brief) exposure, it looks fantastic. I've poked about just a little bit at making a version of simple-v-table that takes on-change handlers in the column descriptions (I use them to set up re-frame dispatch calls that modify the part of the re-frame table that I've used as the :model for the simple-v-table) and then in render-row, if the column has an on-change handler I create an input-text field instead of just the content. So far it's been encouraging enough that I think it might be made reasonably general.

Does it sound like the sort of change that day8/re-com might be open to including?

I am quite new to both re-com and re-frame, so if this is an obviously bad idea and anyone has the bandwidth to let me know why that would also be appreciated :)

**Comment by mike-thompson-day8** [comment]
I'll leave it with @Gregg8 to comment on editting.  But he is away for a few days, so there'll be a delay

**Reply by esnyder** [reply]
Thanks for letting me know. (And for all the work on re-com, it's much appreciated!)
